### PR TITLE
only allow target gene for deblur

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: qp_deblur/deblur.py [hunk 1]
```diff
@@ -373,6 +373,9 @@ def deblur(qclient, job_id, parameters, out_dir):
     prep_info = qclient.get(
         '/qiita_db/prep_template/%s/' % artifact_info['prep_information'][0])
     df = pd.read_csv(prep_info['prep-file'], sep='\t')
+    if prep_info['data_type'] not in {'16S', '18S', 'ITS'}:
```
Comment (wasade) [inline qp_deblur/deblur.py:376]: This variable doesn't look like it exists in any of the AGP preps. Is it common and controlled in Qiita?

Comment (wasade) [reply qp_deblur/deblur.py:376]:
```python
In [61]: pts = qdb.study.Study(10317).prep_templates()

In [62]: obs = set()
    ...: for pt in pts:
    ...:     df = pt.to_dataframe()
    ...:     if 'data_type' in df.columns:
    ...:         obs.update(set(df['data_type']))
    ...:     else:
    ...:         print("Missing: %d" % pt.id)
... output cut ...
In [63]: obs
Out[63]: set()

```

Comment (antgonza) [reply qp_deblur/deblur.py:376]: Yup, given by: https://github.com/biocore/qiita/blob/master/qiita_db/handlers/prep_template.py#L77, which is called 2 lines above.

Comment (wasade) [reply qp_deblur/deblur.py:376]: Oh I see, thanks!

File: qp_deblur/deblur.py [hunk 2]
```diff
@@ -373,6 +373,9 @@ def deblur(qclient, job_id, parameters, out_dir):
...
+        error_msg = ('deblur was developed only for target gene data')
```
Comment (sjanssen2) [inline qp_deblur/deblur.py:377]: is `target gene data` a good name? How about "amplicon sequencing data" it might be closer to the wet lab vocabulary

Comment (sjanssen2) [inline qp_deblur/deblur.py:377]: we maybe also want to give some explanation? `... as it assumes almost all sequences map to the same genomic region.`

Comment (antgonza) [reply qp_deblur/deblur.py:377]: Is this assumption actually in the code or somewhere? I don't know ...

**Comment by ElDeveloper** [comment]
Thanks everyone!

**Pull request merged**